src/tutorials/selfhosting/selfhost-ubuntu-sources-en.md

```diff
@@ -0,0 +1,524 @@
+    echo "deb [signed-by=/usr/share/keyrings/couchdb-archive-keyring.gpg] https://apache.jfrog.io/artifactory/couchdb-deb/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/couchdb.list >/dev/null
+
+Install Couchdb
+
+    sudo apt update
+    sudo apt install -y couchdb
+
+Duing CouchDB installation, choose `Standalone` mode and define admin password.
+
+Validate CouchDB is working
+
+    curl http://localhost:5984/
+    {"couchdb":"Welcome","version":"3.2.1","git_sha":"244d428af","uuid":"f7b83554fa2eb43778963d18a1f92211","features":["access-ready","partitioned","pluggable-storage-engines","reshard","scheduler"],"vendor":{"name":"The Apache Software Foundation"}}
+
+Finally, create a database user and password for  cozy-stack
+
+    read -p "Couchdb password for cozy user: " -r -s COUCH_PASS
+    curl -fsX PUT -u "admin:adminpwd" "http://localhost:5984/_node/couchdb@127.0.0.1/_config/admins/cozy" --data "\"${COUCH_PASS}\""
+
+In this command line, `adminpwd` should be replaced by CouchDB admin password you defined during CouchDB installation.
+
+# NodeJS
+
+To be able to run cozy connectors and gather all your data, cozy-stack needs NodeJS version 12 or 16. This documents gives instructions to instal NodeJS 16.
+
+Configure NodeJS 16 package repository
+
+    KEYRING=/usr/share/keyrings/nodesource.gpg
+    curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | sudo tee "$KEYRING" >/dev/null
+    echo "deb [signed-by=$KEYRING] https://deb.nodesource.com/node_16.x $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/nodesource.list >/dev/null
+    echo "deb-src [signed-by=$KEYRING] https://deb.nodesource.com/node_16.x $(lsb_release -sc) main" | sudo tee -a /etc/apt/sources.list.d/nodesource.list >/dev/null
+
+Install NodeJS
+
+    sudo apt update
+    sudo apt install -y nodejs
+    sudo ln -s /usr/bin/node /usr/bin/nodejs
+
+# Go
+
+cozy-stack is developped in Go language so we need to install the go compiler to be able to compile cozy-stack sources.
+
+    wget -O /tmp/go1.17.3.linux-amd64.tar.gz https://go.dev/dl/go1.17.3.linux-amd64.tar.gz
+    sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzvf /tmp/go1.17.3.linux-amd64.tar.gz
+    echo "export PATH=\"\$PATH:/usr/local/go/bin\"" | sudo tee /etc/profile.d/golang.sh > /dev/null
+    source /etc/profile.d/golang.sh
+
+Test Go installation is fine with
+
+    go version
+    go version go1.17.3 linux/amd64
+
+# Cozy-stack
+
+First, install requirements
+
+    sudo apt install -y imagemagick libprotobuf-c1 fonts-lato
+
+Get the source code
+
+    sudo apt install -y git
+    sudo git clone https://github.com/cozy/cozy-stack.git /opt/cozy-stack
+
+Then compile the program
+
+    cd /opt/cozy-stack
+    scripts/build.sh release $(go env GOPATH)/bin/cozy-stack
+
+The compilation generate a binary file under `$GOPATH/bin/cozy-stack`
+
+You can test it with :
+
+    $(go env GOPATH)/bin/cozy-stack version
+    1.5.0-5-gcbdf012d
```

Comment:
nit: Since we install from https://go.dev/dl/go1.17.3.linux-amd64.tar.gz, the version should be 1.17.3
```suggestion
   1.17.3
```